03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+
+const connectDB = (url) => {
+    return mongoose.connect(url, {
+        useNewUrlParser: true,
+        useCreateIndex: true,
+        useFindAndModify: false,
+        useUnifiedTopology:true,
+    })    
```

Comment:
Once your packages are updated, you'll want to remove these object properties since none are available anymore. You can still pass the object, but doing the below is fine:
`return mongoose.connect(url);`